docs/about.rst

```diff
@@ -29,7 +29,7 @@ paths can be constructed and modified on these views. Exchange through a waypoin
 MSS is a client/server application written in the language **Python**. This is used to create flight plans and
 discuss these with pilots of research aircraft. The flight path is designed by local and remote scientists based
 on meteorologic forecast data on the QT5 MSS gui application.
-A `publication by Rautenhaus et al. <http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012.pdf>`_
+A `publication by Rautenhaus et al. <https://gmd.copernicus.org/articles/15/8983/2022/gmd-15-8983-2022.pdf>`_ (for the previous release click `here <http://www.geosci-model-dev.net/5/55/2012/gmd-5-55-2012.pdf>`_)
```

Comment:
Marc and me are not the same person. So you can't replace it that way. And we  want both academic papers cited in other academic scientific work. Scientist who use our software can cite our work.  There are sources on the net abtout "why are citations are important".

You could change the text by e.g. Two publications are available, one by Rautenhaus et al. .... and one Bauer et al. ... which describes the software.

Reply:
@ReimarBauer Hey, appreciate the info, also I have made a commit following the changes you suggested please take a look.